package.json

```diff
@@ -40,7 +39,7 @@
     "jsdom": "3.0.0",
     "jshint": "^2.6.3",
     "react": "^0.14.0",
-    "react-router": "^1.0.3",
+    "react-router": "^2.4.0",
```

Comment:
@JustinMGaiam Thanks for the error output. I've managed to reproduce it myself in the example app by installing `react-router@1`.

The error was caused by not having a fallback for `react-router` v1 when I bumped `react-router` to v2 in #148.

I've made a fix in this [commit](https://github.com/paypal/react-engine/pull/159/commits/8fd56aa95d9e2ec8ec60c4b542f6a6cce60e91bc). Waiting for @samsel to review it.